lib/utils/schema/schema.rb

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env ruby
```

Comment:
Sure, good point! Also valid since it's still in `lib/utils` instead of some `extras` folder, so I fully 👍 this argument